lightning-invoice/src/utils.rs

```diff
@@ -400,6 +401,9 @@ fn filter_channels(channels: Vec<ChannelDetails>, min_inbound_capacity_msat: Opt
 		if channel.inbound_capacity_msat >= min_inbound_capacity {
```

Comment:
Should we still return an empty vec if any public channel exists (5 lines up), since the public channels may be offline now?